### PR TITLE
Update so the exe name is jbrowse-desktop

### PR DESCRIPTION
**Pull request by cmdcolin**
This produces an exe name that is jbrowse-desktop on all platforms

I think that it is better this way because the "JBrowse 2" exe name is somewhat awkward on linux for example, and could be confused with jbrowse CLI

Also gives the package name a @gmod namespace

### CODE REVIEW

File: products/jbrowse-desktop/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "jbrowse-desktop",
-  "productName": "JBrowse 2",
+  "name": "@gmod/jbrowse-desktop",
+  "productName": "jbrowse-desktop",
```
Comment (garrettjstevens) [inline products/jbrowse-desktop/package.json:3]: I can't remember what this field is used for, but it doesn't seem to change the executable name. I think you also have to change it in `build.productName` and `build.linux.executableName`.

**Review by garrettjstevens — COMMENTED** [review]
I think I'd prefer the name "JBrowse Desktop" over "jbrowse-desktop", since it's a displayed name.

**Comment by rbuels** [comment]
Icons, etc should just have "JBrowse 2" on them, but I agree `jbrowse-desktop` is a good exe name

**Comment by garrettjstevens** [comment]
> Icons, etc should just have "JBrowse 2" on them, but I agree `jbrowse-desktop` is a good exe name

Linux is the only one right now that has an "executable name," Mac and Windows just have the "product name," which is what appears on icons (and Linux doesn't currently have an icon). So maybe we just change it in `build.linux.executableName` and leave the others as "JBrowse 2".

**Comment by cmdcolin** [comment]
Made it so that it manually specifies the artifact names so we get

jbrowse-desktop-{version}-win/mac/linux.zip for all three

And then also has jbrowse-desktop executable name on linux

and then also renames the package to @gmod/jbrowse-desktop

File: products/jbrowse-desktop/package.json (hunk 1 above)
Comment (cmdcolin) [reply products/jbrowse-desktop/package.json]: I proposed maybe removing this if it is unused?

**Comment by cmdcolin** [comment]
Random other things

* Bump to electron 9.1.1 in electronVersion since that reflects the current yarn.lock
* Remove packageName

File: products/jbrowse-desktop/package.json [hunk 2]
```diff
@@ -69,14 +68,16 @@
...
     "build": "rescripts build",
-    "build-electron": "yarn build && electron-builder build --windows --mac --linux",
-    "build-electron:win": "yarn build && electron-builder build --windows",
-    "build-electron:mac": "yarn build && electron-builder build --mac",
-    "build-electron:linux": "yarn build && electron-builder build --linux",
-    "build-electron-unpacked": "yarn build && electron-builder build --windows --mac --linux --dir",
-    "build-electron-unpacked:win": "yarn build && electron-builder build --windows --dir",
-    "build-electron-unpacked:mac": "yarn build && electron-builder build --mac --dir",
-    "build-electron-unpacked:linux": "yarn build && electron-builder build --linux --dir",
+    "prebuild-electron": "yarn build",
```
Comment (garrettjstevens) [inline products/jbrowse-desktop/package.json:71]: This only runs before `build-electron` and not e.g. `build-electron:linux`

File: products/jbrowse-desktop/package.json [hunk 3]
```diff
@@ -92,23 +93,26 @@
   },
   "homepage": "./",
   "build": {
-    "electronVersion": "7.1.0",
+    "electronVersion": "9.1.1",
```
Comment (garrettjstevens) [inline products/jbrowse-desktop/package.json:96]: Maybe we should pin this to a specific version in package.json so it doesn't get out of sync

File: products/jbrowse-desktop/package.json (hunk 2 above)
Comment (cmdcolin) [reply products/jbrowse-desktop/package.json]: darn thought i was being clever. reverted

File: products/jbrowse-desktop/package.json (hunk 3 above)
Comment (cmdcolin) [reply products/jbrowse-desktop/package.json:94]: went ahead and did this, but in general I don't like pinning versions. it seemed like the ideal is it gets electron version from package.json but couldn't get this to work

**Review by garrettjstevens — APPROVED** [review]

**Pull request merged**